widgets/src/frame.rs

```diff
@@ -741,10 +711,6 @@ impl Frame {
         self.area
     }
     
-    pub fn draw(&mut self, cx: &mut Cx2d,) -> WidgetDraw {
-        self.draw_walk(cx, self.get_walk())
-    }
-    
```

Comment:
I believe that we're removing this because the code using it in FrameRef is commented? Should we either delete that code also or keep both?

Reply:
Good call.. I'm removing the related commented code